app/scripts/controllers/app-state.js

```diff
@@ -28,9 +28,11 @@ export default class AppStateController extends EventEmitter {
       preferencesStore,
       qrHardwareStore,
       messenger,
+      extension,
     } = opts;
     super();
 
+    this.extension = extension;
```

Comment:
Just so I'm clear this change was required for mocking purposes? And is probably clean up since want these to be browser agnostic in the future?

Reply:
correct 💪